dist/array_extensions.js

```diff
@@ -0,0 +1,44 @@
+(function (Array) {
+	const EACH_ERROR = 'Function Each already exist';
+	const WHERE_ERROR = 'Function Where already exist';
+
+	var extensions = {
+		'each': { method: each, error: EACH_ERROR },
+		'where': { method: where, error:WHERE_ERROR}
+	};
+
+	function each(callback) {
```

Comment:
I would advice to take advantage of hoisting and early extend `Array`'s prototype.